cmd/dep/testdata/govendor/golden.txt

```diff
@@ -0,0 +1,7 @@
+Detected govendor configuration files...
+Converting from vendor.json...
+Warning: Not able to convert ignoring of build tag 'test'
```

Comment:
The warning should be indented with two spaces, so that it falls under the heading "Converting from vendor.json...".